### DISCUSSION TITLE
Unwrapping

### DISCUSSION THREAD

**Discussion by rofiaini** [gmtsar/gmtsar#910] [Q&A]
Hi everyone, I want to ask about the unwrapping process because of hardware limitations so it takes a long time. from tutorial, there is option "snaphu_interp.csh 0.10 40 1000/3000/24000/27000" for clip according AOI. My questions are:
1. How to get min/max azzimut and range? I already install ncview, which file should i look into? when I look into phase.pdf in F1 and F2 have same coordinate range and azimuth.
2. that I understand from the script snaphu_interp.csh, there is include landmask processing, am I right?
3. to run snaphu_interp.csh, Is it done manually in each interferogram folder?

**Comment by rofiaini** [comment]
here is one of my unwraped result, 
whether the results are good enough?

**Reply by Xiaohua-Eric-Xu** [reply]
Please share the image of your unwrapped phase. :)

**Comment by Xiaohua-Eric-Xu** [comment]
You'll need to check things in the merge directory. i.e. phasefilt.grd in the merge directory.  
snaphu_interp.csh uses landmask_ra.grd if there is one. However, you'll need to produce it with landmask.csh first. 
You can run in every directory or just call unwrap_parallel.csh. But you may need to write a simple unwrap_intf.csh first in your working directory that goes to the target directory, do the unwrapping and come back to your working directory. e.g.
```
cd $1
snaphu_interp.csh 0.0001 0 
cd ..
```
